storage/objects/objects_test.go

```diff
@@ -173,6 +173,21 @@ func TestObjects(t *testing.T) {
 		t.Errorf("contents = %q; want %q", got, want)
 	}
 
+	t.Run("setMetadata", func(t *testing.T) {
+		bkt := client.Bucket(bucket)
+		obj := bkt.Object(object1)
```

Comment:
bkt, obj because without it we have a problem:
=== RUN   TestObjects/setMetadata
    objects_test.go:187: object content = ""; want "value"